src/button-group.button.styles.ts

```diff
@@ -24,31 +24,33 @@ export default [
 
     :host(:last-of-type) .component.horizontal {
       border-end-end-radius: 0.6875rem;
+      border-inline-end: none;
       border-start-end-radius: 0.6875rem;
     }
 
     :host(:last-of-type) .component.vertical {
+      border-block-end: none;
       border-end-end-radius: 0.6875rem;
       border-end-start-radius: 0.6875rem;
     }
 
     .component {
       align-items: center;
       appearance: none;
-      background-color: var(--glide-core-surface-page);
+      background-color: var(--glide-core-color-static-surface-container);
+      color: var(--glide-core-color-static-text-default);
       cursor: pointer;
       display: flex;
-      font-family: var(--glide-core-font-sans);
-      font-size: var(--glide-core-body-md-font-size);
-      font-style: var(--glide-core-heading-xxs-font-style);
```

Comment:
Removed all `font-style` as `normal` is the default and we have no need for any other options.